app/controllers/articles_controller.rb

```diff
@@ -2,7 +2,9 @@ class ArticlesController < ApplicationController
   before_action :find_article, except: [:create]
 
   def create
+    params[:article][:categories].shift
```

Comment:
What does this do for you?

Reply:
The params returns a array with two indexes. The first one is always empty. 
``` params[:article][:categories] = ["", "id_category"] ``` 
We tried a couple of was to remove it.

Reply:
I don’t understand

Reply:
If we loop ``` params[:article][:categories]  ``` as is, we will have a empty string in our ```article.categories```. 
Then we try to id that empty string to display the categories it thorws an error. 
We remove that empty string with the command ``` params[:article][:categories].shift ```
Do you have a suggestion on another solution for this issue?